llumnix/llm_engine_manager.py

```diff
@@ -248,28 +255,109 @@ async def _migrate(self) -> None:
             logger.error("unexpected exception occurs: {}".format(e))
             logger.error("exception traceback: {}".format(traceback.format_exc()))
 
+    async def rebuild_migrate_backend(self) -> None:
+        # Wait for all instances to finish migration
+        while any(self.instance_migrating.values()):
+            await asyncio.sleep(0.1)
+
+        # During rebuilding migrate backend, disable migrate
+        origin_config = self.enable_migrate
+        self.enable_migrate = False
```

Comment:
do we need a lock to protect `self.enable_migrate`?

Reply:
it seems that `self.enable_migrate` will be accessed in different corountines

Reply:
In async actors, only one task can be running at any point in time (though tasks can be multi-plexed). There will be only one thread in AsyncActor! 

So we don't need lock for self.enable_migrate